lib/update-fetch.js

```diff
@@ -2,8 +2,8 @@
 var path = require('path');
 
 // Third Party Dependencies
-var gunzip = require('zlib').createGunzip();
 var extract = require('tar-stream').extract();
+var gunzip = require('zlib').createGunzip();
```

Comment:
Yeah, but I thought I'd done this in another commit on another branch—I must have missed it. I will undo this and make the change elsewhere